java-squid/src/main/java/org/sonar/plugins/java/api/IssuableSubscriptionVisitor.java

```diff
@@ -27,10 +27,18 @@
 
 public abstract class IssuableSubscriptionVisitor extends SubscriptionVisitor {
 
+  /**
+   * @deprecated use reportIssue instead to benefit from precise issue location.
+   */
+  @Deprecated
   public void addIssue(Tree tree, String message) {
```

Comment:
![INFO](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-info.png) Do not forget to remove this deprecated code someday. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://dory.sonarsource.com/coding_rules#rule_key=squid%3AS1133)